jest.config.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
   ],
   moduleNameMapper: {
     "\\.(jpg|jpeg|png|gif|eot|otf|svg|ttf|woff|woff2)$": "<rootDir>/components/test/fileMock.ts",
-    "\\.(css|less)$": "identity-obj-proxy",
+    "\\.(css|less|scss)$": "identity-obj-proxy",
```

Comment:
Did an issue with this come up in this PR? I don't see any scss files added.

Reply:
Yeah, so I'm depending on a web component which depends on `brave/components/web-components/app.global.scss` which  was making `jest` very unhappy.